docs/AboutUs.md

```diff
@@ -39,14 +39,14 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 * Role: Developer
 * Responsibilities: Code Quality
 
-### Johnny Doe
+### Tan Hong Liang
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/tanhl2000.png" width="200px">
 
-[[github](http://github.com/johndoe)] [[portfolio](team/johndoe.md)]
+[[github](http://github.com/tanhl2000)] [[portfolio](team/tanhl2000.md)]
 
-* Role: Developer
-* Responsibilities: Data
+* Role: Integration
+* Responsibilities: Merging team commits
```

Comment:
Hey, role should be Developer for all of us except Marcus & then responsibility should be Integration
```suggestion
* Role: Developer
* Responsibilities: Integration
```